front/src/app/routes/routes.ts

```diff
@@ -288,10 +288,6 @@ export const { RouteProvider, useRoute, routes } = createRouter({
     () => `/${frontRoutes.magicLinkRenewal}`,
   ),
   search: defineRoute(searchParams, () => `/${frontRoutes.search}`),
```

Comment:
pour avoir le redirect ici (pour éviter les config nginx), on peut faire : 

```suggestion
  search: defineRoute(searchParams, () => [`/${frontRoutes.search}`, "/recheche-diagoriente"]),
```

(pas de préférence, je te laisse le choix)